tests/metrics_test.go

```diff
@@ -167,87 +158,48 @@ var _ = Describe("Metrics", func() {
 	})
 
 	Context("SSP metrics", func() {
-		var originalVersion string
-
-		waitForVersionUpdate := func(version string) {
-			Eventually(func() error {
-				rsList := &apps.ReplicaSetList{}
-				err := apiClient.List(ctx, rsList, client.InNamespace(sspDeploymentNamespace),
-					client.MatchingLabels{"name": "ssp-operator"})
-				Expect(err).ToNot(HaveOccurred())
-
-				for _, rs := range rsList.Items {
-					for _, env := range rs.Spec.Template.Spec.Containers[0].Env {
-						if env.Name == common.OperatorVersionKey &&
-							env.Value == version &&
-							*rs.Spec.Replicas > 0 &&
-							rs.Status.ReadyReplicas == *rs.Spec.Replicas {
-							return nil
-						}
-					}
-				}
-
-				return fmt.Errorf("operator version not yet updated")
-			}, env.ShortTimeout(), time.Second).Should(Succeed())
-		}
+		var templates *templatev1.TemplateList
```

Comment:
Only the first template is used in the tests. Can you change the type of this variable to `*templatev1.Template`?